config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -55,10 +56,10 @@ presubmits:
         image: gcr.io/k8s-testimages/kubekins-e2e:v20200807-c10eea0-master
         resources:
           limits:
-            cpu: 4
+            cpu: 6
```

Comment:
Is 4 really not enough?

Do we allocate 6 because of bazel overhead? If so, might be good to add an explicit comment (here an n other places).

Reply:
Yes, it's because of bazel. We could go for lower CPU, but that would increase a duration of presubmit jobs, so probably we want to optimize for time, rather than resource consumption.

I will add comment